Source/WebCore/platform/graphics/mac/controls/WebControlView.mm

```diff
@@ -130,6 +130,11 @@ - (NSView *)_focusRingClipAncestor
     return self;
 }
 
+- (NSWindow *)_viewRoot
```

Comment:
should we declare `_viewRoot` as SPI?

Reply:
It's IPI that is not explicitly called, similar to `_automaticFocusRingDisabled`, so I have omitted it from SPI headers.